addons/portal/wizard/portal_wizard.py

```diff
@@ -89,6 +98,25 @@ class PortalWizardUser(models.TransientModel):
     login_date = fields.Datetime(related='user_id.login_date', string='Latest Authentication')
     is_portal = fields.Boolean('Is Portal', compute='_compute_group_details')
     is_internal = fields.Boolean('Is Internal', compute='_compute_group_details')
+    email_validity_state = fields.Selection(
+        [('valid', 'Valid Email'), ('invalid', 'Invalid Email'), ('already_registered', 'Email Already Registered')],
+        string='Email Status', compute='_compute_email_validity_state', default='valid')
+
+    @api.depends('email')
+    def _compute_email_validity_state(self):
+        for portal_wizard_user in self:
+            email_normalized = email_normalize(portal_wizard_user.email)
+            if not email_normalized:
+                portal_wizard_user.email_validity_state = 'invalid'
+            else:
+                user = self.env['res.users'].sudo().with_context(active_test=False).search([
+                    ('id', '!=', portal_wizard_user.user_id.id),
+                    ('login', '=ilike', portal_wizard_user.email),
```

Comment:
Looks like the code was previously testing against "email_normalized" and now it's testing against the raw email input?